scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/graphql/GraphQLParsersTest.java

```diff
@@ -28,15 +28,7 @@ public static Stream<Arguments> parseGraphQLInterval() {
         Arguments.of("322:21:15.", parseDurationISO8601("PT322H21M15S")),
         Arguments.of("322:21:15", parseDurationISO8601("PT322H21M15S")),
         Arguments.of("+322:21:15", parseDurationISO8601("PT322H21M15S")),
-        Arguments.of("-322:21:15", parseDurationISO8601("PT-322H-21M-15S")),
-        Arguments.of("21:15.111", parseDurationISO8601("PT21M15.111S")),
-        Arguments.of("+21:15.111", parseDurationISO8601("PT21M15.111S")),
-        Arguments.of("-21:15.111", parseDurationISO8601("PT-21M-15.111S")),
-        Arguments.of("15.111", parseDurationISO8601("PT15.111S")),
-        Arguments.of("15.", parseDurationISO8601("PT15S")),
-        Arguments.of("15", parseDurationISO8601("PT15S")),
-        Arguments.of("-15", parseDurationISO8601("PT-15S")),
-        Arguments.of("+15", parseDurationISO8601("PT15S"))
```

Comment:
Why were these parse tests removed?

Reply:
Because they are not valid wrt to the PGInterval class and do not get parsed correctly. 
Here is a link to the discussion https://jpl.slack.com/archives/CJ07FGRPH/p1690482612292829 for more details.